### PR TITLE
Topotato: Vagrant Support

### PR DESCRIPTION
**Pull request by eznix86**
Purpose:
=======

If you have linux, you can create multiple VM in one spot, and just run the same commands again and again, for now it uses Virtualbox, but there can be many providers, we can do dynamic stuff later, but here is a sneak peak of what it should be working.

Known issue:
=========
Due to the command unshare and no uid mapping, we get the nobody,nogroup permission set on files which are created by topotato

Installation
=========

- Download Vagrant (https://developer.hashicorp.com/vagrant/downloads)
- Ensure that the `vagrant` cli is ready. (type `vagrant` command)
- Ubuntu Host
  - Install virtualbox provider `sudo apt install virtualbox` 
  - Install vb-guest: `vagrant plugin install vagrant-vbguest`
  - Do `vagrant up --provider virtualbox` 

Some commands (see Makefile)
- make run

### CODE REVIEW

File: vm/ubuntu/install.sh [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+    --enable-logfile-mask=0640 \
+    --enable-snmp=agentx \
+    --enable-multipath=64 \
+    --enable-user=frr \
+    --enable-group=frr \
+    --enable-vty-group=frrvty \
+    --with-pkg-git-version \
+    --with-pkg-extra-version=-MyOwnFRRVersion
+make
+sudo make install
```
Comment (eqvinox) [inline vm/ubuntu/install.sh:54]: topotato should work without FRR being installed, everything FRR-related from here should be unnecessary, except the `mkdir /etc/frr` (which is an open/known problem)

(the sysctl/module bits stay)

Comment (eqvinox) [reply vm/ubuntu/install.sh:54]: actually except keep this one line from 53 → 63:
```
sudo install -m 775 -o frr -g frrvty -d /etc/frr
```

Comment (eznix86) [reply vm/ubuntu/install.sh]: fixed

**Comment by eznix86** [comment]
ready to merge

**Pull request merged**